packages/aws-rfdk/lib/core/test/health-monitor.test.ts

```diff
@@ -135,7 +135,23 @@ describe('HealthMonitor', () => {
       KeyPolicy: {
         Statement: [
           {
-            Action: 'kms:*',
+            Action: [
```

Comment:
It's curious to me that this changed. Implies that we're now going down the `addLegacyAdminPolicy()` branch of https://github.com/aws/aws-cdk/blob/6359c12e3242e23d9b3bf0a42cac7c361c8d4d8a/packages/%40aws-cdk/aws-kms/lib/key.ts#L637

Looks like it's safe to update the policy ( https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-kms-key.html#cfn-kms-key-keypolicy ) -- doesn't cause a replacement.

I'd guess this is related to `@aws-cdk/aws-kms:defaultKeyPolicies` feature flag (what version of CDK was that introduced in?). Since the `trustAccountIdentities` flag was removed from this KMS key, this implies to me that the tests aren't being run with that feature flag set.

Reply:
I've changed this back to how it was originally, I believe you're right that it seems we aren't applying the feature flags to the tests but that seems like something we can deal with if/when we upgrade to CDK v2